docs/docs/llms/deployments/index.mdx

```diff
@@ -1469,7 +1469,7 @@ and a config class that inherits from `mlflow.gateway.base_models.ConfigModel`.
 import os
 from typing import AsyncIterable
 
-from pydantic import validator
+from mlflow.utils import validator
```

Comment:
isn't it clearer to import from `mlflow.utils.pydantic`? or `mlflow.utils.pydantic_utils`?